### PR TITLE
feat: add a command to set player's locale

### PR DESCRIPTION
**Pull request by Tomut0**

### CODE REVIEW

File: src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/general/GeneralCommands.java [hunk 1]
```diff
@@ -65,6 +62,17 @@ public void main(CommandSender sender) {
         }
     }
 
+    @Subcommand("%locale")
+    @CommandPermission("simpleclans.anyone.locale")
+    @Description("{@@command.description.locale}")
+    @CommandCompletion("@locales")
+    public void locale(ClanPlayer cp, @Name("locale") String locale) {
```
Comment (RoinujNosde) [inline src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/general/GeneralCommands.java]: `@Values("@locales")` to validate the input based on the completion list

**Review by RoinujNosde — CHANGES_REQUESTED** [review]

File: src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/general/GeneralCommands.java (hunk 1 above)
Comment (Tomut0) [reply src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/general/GeneralCommands.java]: Nice 🤔 👍🏻

Comment (RoinujNosde) [reply src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/general/GeneralCommands.java]: No, you

File: src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/staff/StaffCommands.java [hunk 2]
```diff
@@ -373,4 +373,16 @@ public void rename(CommandSender sender, @Name("clan") ClanInput clanInput, @Nam
 
         ChatBlock.sendMessageKey(sender, "you.have.successfully.renamed.the.clan", clanName);
     }
+
+    @Subcommand("%locale %set")
```
Comment (RoinujNosde) [inline src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/staff/StaffCommands.java]: %mod prefix?

**Review by RoinujNosde — CHANGES_REQUESTED** [review]

File: src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/staff/StaffCommands.java (hunk 2 above)
Comment (Tomut0) [reply src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/staff/StaffCommands.java]: > %mod prefix?

My bad, forgot about it 😅

**Review by RoinujNosde — APPROVED** [review]

**Review by SrBedrock — APPROVED** [review]

**Pull request merged**